src/Roassal3-Chart-Examples/RSBarChartExample.class.st

```diff
@@ -230,44 +230,44 @@ RSBarChartExample >> example06BarAnimation [
 RSBarChartExample >> example07DoubleBar [
 
 	<script: 'self new example07DoubleBar open'>
-	| c classes x1 x2 y plot title |
-	c := RSCompositeChart new.
+	| classes x1 x2 y plot title |
 	classes := RSShape withAllSubclasses asOrderedCollection sorted: [
 		           :a
 		           :b | a name < b name ].
 	x1 := classes collect: #numberOfMethods.
 	x2 := classes collect: #numberOfVariables.
 	y := 1 to: classes size.
-	c add: (plot := RSDoubleBarPlot new
-			         x1: x1 x2: x2 y: y;
-			         yourself).
+	plot := RSDoubleBarPlot new x1: x1 x2: x2 y: y.
 
-	c horizontalTick numberOfTicks: 10.
-	c addDecoration: (RSHorizontalTopTick new
+	plot horizontalTick
+		numberOfTicks: 10;
+		labelConversion: #asInteger.
```

Comment:
Is the labelConversion: needed here and below??

Reply:
It's just for the example, to be more readable. With the default float values it's a bit harder to read.
I think it's better like this:
![image](https://github.com/ObjectProfile/Roassal3/assets/124769707/c356bdea-34bd-4b24-8a4d-4cd5fb40d344)

than like this:
![image](https://github.com/ObjectProfile/Roassal3/assets/124769707/8c940b5c-ce20-418a-aaea-8b35f5137b0b)

And again it's just for the clarity of the example